install/ci-vm/ci-linux/ci/runCI

```diff
@@ -142,8 +86,14 @@ if [ -x "ccextractor" ]; then
 	executeCommand ${tester} --entries "${testFile}" --executable "${dstDir}/linux/ccextractor" --tempfolder "${tempFolder}" --timeout 3000 --reportfolder "${reportFolder}" --resultfolder "${resultFolder}" --samplefolder "${sampleFolder}" --method Server --url "${reportURL}"
 	postStatus "completed" "Ran all tests"
 	sendLogFile
+	# Delete build files after tests are complete
```

Comment:
Strictly speaking this is unnecessary as we restart from a clean KVM image point each time.